bin/ch-grow.py.in

```diff
@@ -191,7 +190,10 @@ environment variables:
    state = State()
 
    parser = lark.Lark(GRAMMAR, parser="earley", propagate_positions=True)
-   text = open(cli.file, "rt").read()
+   if (cli.file == "-"):
```

Comment:
What motivated this change?

Reply:
Good question. It's common for UNIX apps to accept `-` (single hyphen) as a filename indicating standard input or output. `docker` does this, and I added it in this PR for parity to make the tests pass.

Reply:
Is this argument covered in the copy/test.bats file?

Reply:
It's in one of the Bats files; not sure which one.